ConsoleGame.java

```diff
@@ -3,18 +3,23 @@
 public class ConsoleGame implements GameType {
 
     private final Writer writer;
-    private InputStream stream;
     private BufferedReader reader;
 
     public ConsoleGame(Writer writer, InputStream stream) {
         this.writer = writer;
-        this.stream = stream;
+        InputStream inputStream = stream;
         reader = new BufferedReader(new InputStreamReader(stream));
     }
 
     @Override
     public void drawBoard(Board board) {
-        print("|" + board.getState().substring(0, 3) + "|\n|" + board.getState().substring(3, 6) + "|\n|" + board.getState().substring(6, 9) + "|\n");
+        for (int i=0; i < board.dimension * board.dimension; i+=board.dimension) {
```

Comment:
work on the UX of the board.  I think at the moment it will look like: 
|xxxx|
|oooo| etc..  

Try to make it a bit nicer with some spacing and maybe some colour for each mark.